awsx-classic/tslint.json

```diff
@@ -28,7 +28,41 @@
         "member-access": false,
         "member-ordering": [
             true,
```

Comment:
Could we just remove`member-ordering` alltogether. In my opinion it adds nothing but pain. And also remove`ordered-imports` while we're there too?

Reply:
Opening a new PR.